src/domain/orders/details/returns/index.js

```diff
@@ -30,12 +32,27 @@ const ReturnMenu = ({ order, onReturn, onDismiss, toaster }) => {
 
   const [allItems, setAllItems] = useState([])
 
+  const [showCustomPrice, setShowCustomPrice] = useState({
+    standard: false,
```

Comment:
Is this used?

Reply:
Yes - used to either display the custom price for shipping or not (and determining to include it in the request)

Reply:
I am referring to the `standard` property for all comments to the PR. Looking at the files, it does not really seem to be used, but can you please confirm? Cause, then i will delete

Reply:
@sebastiannicolajsen